ui/src/app/views/app/component.html

```diff
@@ -1,8 +1,13 @@
-<div>
-  <img class="main-logo" src="assets/images/all-of-us-logo.svg" alt="All of Us"
-    style="width: 18rem;">
-  <img class="portal-logo" src="assets/images/portal-logo.svg" alt="Researcher Portal"
-    style="margin-left: 0.5rem; width: 13rem;">
+<div style="display: flex; justify-content: space-between;">
```

Comment:
Do we have a plan or (ahem) styleguide for when to apply inline styles? My general preference is to avoid this, but there may be good reasons for it sometimes.